packages/css-data/src/__generated__/properties.ts

```diff
@@ -2000,6 +2000,15 @@ export const properties = {
     },
     types: ["offset-path", "coord-box"],
   },
+  offsetPosition: {
```

Comment:
offset-anchor seems missing as well
https://developer.mozilla.org/en-US/docs/Web/CSS/offset-anchor

Reply:
Good catch, but i see `offset-anchor` is marked as `standard` in the `mdn-data`. But inside `mdn-data` it is missing `mdn_url` and so the generator is skipping it. Kinda might need to add a `wild-card` check for this. What's you opinion ?

Any change i make to MDN, they are not merging at the moment.


```json
"offset-anchor": {
    "syntax": "auto | <position>",
    "media": "visual",
    "inherited": false,
    "animationType": "position",
    "percentages": "relativeToWidthAndHeight",
    "groups": [
      "CSS Motion Path"
    ],
    "initial": "auto",
    "appliesto": "transformableElements",
    "computed": "forLengthAbsoluteValueOtherwisePercentage",
    "order": "perGrammar",
    "status": "standard"
  },
```

Reply:
Some day 😄 
for now just add to experimental list